test/tracing/mongodb_test.js

```diff
@@ -99,7 +102,9 @@ describe('tracing/mongodb', function() {
     });
   });
 
-  it('must trace find requests with cursors', function() {
+  // correlating multiple operations (bulk writes, find with getMore) is broken in mongodb APM support since
+  // version 3.0.6, see https://groups.google.com/forum/#!topic/node-mongodb-native/uBae-HO0zw8
```

Comment:
Some time ago, I was contemplating to switch instrumentation to https://github.com/mongodb-js/mongodb-core, i.e. the underlying mongodb driver. Nothing to do right now, but an option to keep in mind. It might be more stable to instrument the methods within that driver.

Reply:
It just seems kind of a shame to resort to monkey patching when this is actually one of the rare libs that offer an official APM interface. Also, the monkey patching approach has its own set of downsides, so I'm not sold on the idea that this move would be beneficial.